### DISCUSSION TITLE
Option to disable Touchpad when lid is closed

### DISCUSSION THREAD

**Discussion by artungonul** [seerge/g-helper#1515] [Q&A]
Hey everyone, thank you all for this app as it has been a life saver for me. My Tuf a15 (2023) wakes itself up from sleep because of random ghost touchpad inputs(Checked from the SleepStudy, happens every 2 minutes). 

Would it be possible to add an option to disable touchpad when lid is closed (or everything to be honest, laptop wakes up from mouse movement even though wake_armed is  NONE)

**Comment by seerge** [comment]
@artungonul hello, I have to admit that it's quite specific request you have :)

I don't catch / listen to any events in the app that fire on the moment lid is closed, so I can't add anything here.

I would advice to reinstall (mb downgrade even) touchpad drivers. Mb that would fix your problems

**Reply by artungonul** [reply]
Thanks for the answer and the suggestion @seerge I'll definitely try that.

**Reply by IceStormNG** [reply]
To add to this: there is no reliable way to detect lid close or opening. As Windows keeps those events for itself but they're not accessible for applications, only for drivers.
I had that issue already when trying to add some detection to my clamshell mode. The Lid open/close detection is super not impossible but super unreliable.
What you could do instead is to change the windows setting to prevent the touchpad from waking your device at all.

**Reply by artungonul** [reply]
Hey @IceStormNG thank you for the additional information and suggestion. At this point, I pretty much gave up on the sleep function and started using hibernate.
